.gitattributes

```diff
@@ -0,0 +1,11 @@
+# We'll let Git's auto-detection algorithm infer if a file is text. If it is,
+# enforce LF line endings regardless of OS or git configurations.
+
+# More about LF vs. CRLF on the link below:
+# https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/
+
+* text=auto eol=lf
```

Comment:
I'm anxious about the `auto` and the need to try and have an exhaustive list of binary files otherwise we might brick a binary file ... that sounds like a very unpleasant experience! For example, we're missing `pdf` below, which I assume that `git` will properly identify as a binary, but it's weird that we would need to specify things like `png` but not `pdf`!? So, I think we should change the experience here. We should add all the files that we know we wnt to have LF line-endings and never possibly brick any files. The failure experience here better, a developer has to come and update the `.gitattributes` file if we had forgotten to add a particular file type, but no files are bricked!

Please also add a comment in here about how to renormalize all files for the inevitable case that we'll need to run that command when one of us adds another text file extension.